_stylesheets/docs.css

```diff
@@ -868,8 +868,29 @@ table > tbody > tr > td > div > div > p > code {
      border: 0;
```

Comment:
These CSS changes can be ignored for now. Waiting for these changes in another PR to merge, but wanted to include them here for the preview build's sake.